en/docs/administer/changing-to-mariadb.md

```diff
@@ -1,85 +1,164 @@
+    you can use MySQL db scripts as an alternative.
+
+### Changing the default datasource
+
+1.  **Minimum Configurations for changing default datasource to MariaDB.**
+ 
+ Configurations can be done by editing the default configurations in `<IS-HOME>/repository/conf/deployment.toml`. 
+ Following are the basic configurations and their descriptions. 
+      <table>
+      <thead>
+      <tr class="header">
+      <th>Element</th>
+      <th>Description</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr class="even">
+      <td><strong>username</strong> and <strong>password</strong></td>
+      <td>The name and password of the database user.</td>
+      </tr>
+      <tr class="even">
+      <td><strong>driver</strong></td>
+      <td>The jdbc driver of the database.</td>
+      </tr>
+      <tr class="even">
+      <td><strong>url</strong></td>
+      <td>The url of the database.</td>
+      </tr>
+      </table>  
+ 
+ A Sample configuration is given below.
+
+   1. `WSO2_IDENTITY_DB` 
+    
+       1. `deployment.toml` Configurations
+
+           ``` toml
+           [database.identity_db]
+            url = "jdbc:mariadb://localhost:3306/project"
+            username = "regadmin"
+            password = "regadmin"
+            driver = "org.mariadb.jdbc.Driver"
+            [database.identity_db.pool_options]
+            maxActive = "80"
+            maxWait = "60000"
+            minIdle = "5"
+            testOnBorrow = true
+            validationQuery="SELECT 1"
+            validationInterval="30000"
+            defaultAutoCommit=false
+           ```
+       
+       1. Executing database scripts.
+        
+          Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the following files, against the database created.
+           
+           - `<IS-HOME>/dbscripts/identity/mysql.sql`
+           - `<IS-HOME>/dbscripts/identity/uma/mysql.sql`
+           - `<IS-HOME>/dbscripts/consent/mysql.sql`
+         
+   2. `WSO2_SHARED_DB`
+        
+       1. `deployment.toml` Configurations
+
+           ``` toml
+           [database.shared_db]
+            url = "jdbc:mariadb://localhost:3306/project"
+            username = "regadmin"
+            password = "regadmin"
+            driver = "org.mariadb.jdbc.Driver"
+            [database.identity_db.pool_options]
+            maxActive = "80"
+            maxWait = "60000"
+            minIdle = "5"
+            testOnBorrow = true
+            validationQuery="SELECT 1"
+            validationInterval="30000"
+            defaultAutoCommit=false
+           ```
+           
+       1. Executing database scripts.
+        
+          Navigate to `<IS-HOME>/dbscripts`. Execute the scripts in the following file, against the database created.
+                      
+           - `<IS-HOME>/dbscripts/mysql.sql`
+           
+   3. If you have a requirement in using workflow feature follow, 
```

Comment:
If you require the workflow feature, follow

Reply:
done